casadm/cas_lib.c

```diff
@@ -928,11 +1128,12 @@ int start_cache(uint16_t cache_id, unsigned int cache_init,
 
 	cmd.cache_id = cache_id;
 	cmd.init_cache = cache_init;
-	strncpy_s(cmd.cache_path_name,
-		  sizeof(cmd.cache_path_name),
-		  cache_device,
-		  strnlen_s(cache_device,
-			    sizeof(cmd.cache_path_name)));
+	name_set_result = set_cache_path(cmd.cache_path_name, cache_device, 
```

Comment:
cache path is required to be provided by id as well, right? so why this convertion to by-id path? I think it be treated the same way as core path is.

Reply:
It's a good idea, I'll do it Your way.